packages/x-date-pickers/src/internals/hooks/validation/models.ts

```diff
@@ -3,12 +3,12 @@
  */
 export interface BaseDateValidationProps<TDate> {
```

Comment:
I'll create a `FutureAndPastValidationProps` to avoid using a date related interface on time components